Formula/n/navidrome.rb

```diff
@@ -0,0 +1,33 @@
+class Navidrome < Formula
+  desc "Modern Music Server and Streamer compatible with Subsonic/Airsonic"
+  homepage "https://www.navidrome.org"
+  url "https://github.com/navidrome/navidrome/archive/refs/tags/v0.51.1.tar.gz"
+  sha256 "fc962e3acbedfad63934eda016d4e380dd3a06b4636f2b1e61ade9700a2addcd"
+  license "GPL-3.0-only"
+  head "https://github.com/navidrome/navidrome.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "node@18" => :build
```

Comment:
any issue tracker for building with latest node?

Reply:
Looks like they recently merged an update to use Node 20, and it already builds fine with that version. https://github.com/navidrome/navidrome/pull/2861
Should I update this PR to use the latest node?

Reply:
yes, please

Reply:
fixed!